Aplib.Tests/Intent/Actions/ActionTests.cs

```diff
@@ -144,13 +146,13 @@ public void IsActionable_QueryIsNotNull_IsActionable()
     public void IsActionable_QueryIsNull_IsNotActionable()
     {
         // Arrange
-        GuardedAction<object> action = new(guard: () => null!, effect: b => { });
+        GuardedAction<IBeliefSet, object> action = new(guard: _ => null!, effect: (_, _) => { });
 
         // Act
-        bool result = action.IsActionable();
+        bool actionable = action.IsActionable(It.IsAny<IBeliefSet>());
```

Comment:
Let us not use the mocking framework in the 'act' section of a test. Create a mock of the `IBeliefSet` in the 'arrange' section.

We want to let the act to seem like a normal method call.